content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Running Kubernetes node components as a non-root user (`KubeletInUserNamespace`)
+content_type: task
+---
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+This document describes how to run Kubernetes node components such as kubelet, CRI, OCI, and CNI
+without root privileges, by using {{< glossary_tooltip text="a user namespace" term_id="userns" >}}
+and `KubeletInUserNamespace` feature gate.
+
+This technique is also known as "Rootless mode".
+
+{{< note >}}
+This document describes how to run Kubernetes node components (and hence pods) a non-root user.
+
+If you are just looking for how to run a pod as a non-root user, see [SecurityContext](/docs/tasks/configure-pod-container/security-context/).
+{{< /note >}}
+
```

Comment:
As this is a task page, it  should have a prerequisites section. That would be where you could list having a Linux computer, or a container running within containerd / Docker / Podman. Plus any other prerequisites (do you need access to configure subuid / subgid?)

See other task pages for an example.

Reply:
updated